.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "4.0"
 before_install:
   - rvm install 2.2.2
```

Comment:
@zurb/yetinauts does anybody know if we need this `rvm install` script for anything besides scss-lint? If that's all it's being used for I think I will just remove it in this PR.